web/satellite/src/components/header/resourcesDropdown/ResourcesDropdown.vue

```diff
@@ -20,12 +23,26 @@ import { RouteConfig } from '@/router';
     },
 })
 export default class ResourcesDropdown extends Vue {
+    
+    private readonly analytics: AnalyticsHttpApi = new AnalyticsHttpApi();
     /**
      * Indicates if current route is onboarding tour.
      */
     public get isOnboardingTour(): boolean {
         return this.$route.path.includes(RouteConfig.OnboardingTour.path);
     }
+
+    public onDocsIconClick(): void {
+        this.analytics.linkEventTriggered(AnalyticsEvent.EXTERNAL_LINK_CLICKED,"https://documentation.storj.io");
```

Comment:
nit - add space after comma (same comment for other IconClick events)

Reply:
Done.